generator/src/main/java/com/intendia/openapi/Main.java

```diff
@@ -6,34 +6,42 @@
 import static com.google.common.base.Strings.nullToEmpty;
 import static java.util.Collections.emptyMap;
 import static java.util.Optional.ofNullable;
-import static java.util.function.Function.identity;
 import static java.util.stream.Collectors.joining;
-import static java.util.stream.Collectors.toMap;
 
 import com.google.common.base.Strings;
 import com.google.common.base.Throwables;
 import com.google.gson.Gson;
+import com.intendia.gwt.autorest.client.AutoRestGwt;
+import com.intendia.gwt.autorest.client.IgnoreRest;
+import com.intendia.gwt.autorest.client.Security;
```

Comment:
oh, but all these annotations are... private annotations that you have created... 😬 before being able to merge this PR we need to see what to do with those annotations

Reply:
I think you wrote this before seeing the pull to autorest...?